src/utils/findpackage/packageunix.rs

```diff
@@ -17,25 +18,41 @@ use super::{
 
 // here is the logic of findpackage on linux
 //
-const PREFIXS: [&str; 2] = ["/usr", "/usr/local"];
+static PREFIXES: LazyLock<Vec<String>> = LazyLock::new(|| {
+    if let Ok(command) = Command::new("cmake").arg("--system-information").output() {
+        let output = String::from_utf8(command.stdout).unwrap();
+        for line in output.lines() {
+            if line.starts_with("CMAKE_SYSTEM_PREFIX_PATH") {
+                // The line looks something like this:
+                // 'CMAKE_SYSTEM_PREFIX_PATH "<prefix>;<prefix>"'
+                let (_, prefix_paths) = line.split_once(" ").unwrap();
+
+                // Strip leading and trailing quotation marks
+                let prefix_paths = prefix_paths
+                    .strip_prefix(r#"""#)
+                    .unwrap()
+                    .strip_suffix(r#"""#)
+                    .unwrap();
+
+                // FIXME: This likely contains duplicate entries of '/usr/local' on most systems
+                // This could be solved by using Itertools::unique()
+                return prefix_paths.split(";").map(String::from).collect();
+            }
+        }
+    }
+
+    // Use some generic prefixes
+    vec!["/usr/local".into(), "/usr".into(), "/".into()]
+});
 
 const LIBS: [&str; 5] = ["lib", "lib32", "lib64", "share", "lib/x86_64-linux-gnu"];
 
-fn get_prefixs() -> Vec<String> {
+fn get_prefixes() -> Vec<String> {
+    let mut prefixes = (*PREFIXES).clone();
     if let Ok(prefix) = std::env::var("PREFIX") {
-        let mut prefixs: Vec<String> = PREFIXS
-            .to_vec()
-            .iter()
-            .map(|prefix| prefix.to_string())
-            .collect();
-        prefixs.push(prefix.to_string());
-        return prefixs;
+        prefixes.push(prefix.to_string());
```

Comment:
if use the prefixes which provided by cmake, we do not need to check the var, because it is already provided by that command.